### ISSUE TITLE
Enter Admin Mode should take user directly to related project/page

### ISSUE THREAD

**Issue by mkgold** [ManifoldScholar/manifold#1960]
## Feature Description
When one clicks Enter Admin Mode, one is always taken to the home dashboard page. This can be frustrating when one is working on a particular project on the front end and wants to make a change -- clicking "enter admin mode" while on a project homepage doesn't take the person directly to that project, but rather to the general admin homepage

Similarly, if one is looking at a particular Page on an installation and decides that editing is needed, one has to navigate from the admin dashboard homepage back over to pages, which can be cumbersome

## *Why* is this feature important? *Who* does it help?
this feature will help anyone who is working on a project or a page and sees a need to edit or change it on the dashboard

## User Stories
I am an editor at a Press and I want to place a resource within the text of a book. I try to add it to the text and realize I have not yet added it as a resource on the dashboard. So, I click "Enter Admin Mode" but then have to find/search the text before I can add the resource

I am the admin of a Manifold instance and I want to add a page to the installation providing directions on how to get in touch with the people running it. I add the page and then go to look at it on the frontend. A few days later, I'm rereading the page on the frontend and realize I need to make an edit. I click "enter admin mode" but then have to find the page in question before i can edit it

**Comment by zdavis** [comment]
I agree that this would be useful, and not particularly difficult to implement. Worst case, the component that renders that button could look at the window location and parse it for project/:id, text/:id, resource_collection/:id, etc. to jump to the appropriate screen in the backend.

Or we might want to create some React context here so that high-level wrapper components could register a backend URL with that context, to be picked up by the admin component.

Would be nice to have the corresponding feature on the backend. Eg, when I click exit backend while editing a project, it takes me to that project page.

**Comment by SMaxOwok** [comment]
What about an "Edit" button only visible to those with permissions somewhere on the pages?  Maybe changing the text of the button to "Admin Dashboard" or something along with that.  

In my mind, taking what Matt is saying into consideration, "Admin Mode" feels like something that changes the _current_ page to be editable as opposed to navigating to a different section of the app.

Along with that, if I'm expecting to enter the backend (not necessarily a specific project/text/etc.), it feels weird that I wouldn't end up at the actual backend entry point.

Just some thoughts to go along with what's been put forward.

**Comment by zdavis** [comment]
During team meeting, consensus is that taking the user to the project in the BE is sufficient. Does not need to be more granular.

**Comment by zdavis** [comment]
@bot accept

**Comment by zdavis** [comment]
Thank you for taking the time to open this feature request. The Manifold team reviewed this issue during our bi-weekly meeting and the consensus is that this feature makes sense and is in keeping with our overall vision for the platform. Moreover, we see this request as a viable candidate for development under our current available funding. We’re adding an “accepted” label to this request to indicate that it’s within scope and possibly within budget.

The next step is for us to estimate the work involved with this and add it to our feature backlog. Our acceptance of the issue is not a promise that it will be implemented. We will balance this request against the other accepted requests and do our best to implement it within our current available funding.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_
